src/main/fc/cli.c

```diff
@@ -1759,6 +1759,81 @@ static void cliSerial(char *cmdline)
     memcpy(currentConfig, &portConfig, sizeof(portConfig));
 }
 
+#ifndef SKIP_SERIAL_PASSTHROUGH
```

Comment:
Yes, I think we should change to `#ifdef USE_SERIAL_PASSTHROUGH` and enable that feature in `common.h` only for targets with enough flash